src/entities/permitHelper/index.ts

```diff
@@ -176,7 +175,9 @@ const signPermit = async (
 const getDomain = async (client: PublicWalletClient, token: Hex) => {
     const [, name, version, chainId, verifyingContract, , ,] =
         await client.readContract({
-            abi: weightedPoolAbi_V3,
+            abi: parseAbi([
+                'function eip712Domain() view returns (bytes1 fields, string name, string version, uint256 chainId, address verifyingContract, bytes32 salt, uint256[] extensions)',
+            ]),
```

Comment:
@brunoguerios

Think this may be better since this was only place `weightedPoolAbi_V3` was being used and it doesn't seem to be directly related to `getDomain` function or `PermitHelper` class

Happy to revert if team prefers tho. Assuming the abi came from a `MockStablePool` deployment?

Reply:
I used the pool abi initially for "simplicity", but I agree it's not pool specific and looks better with the explicit abi like this 👍